metadata-service/configuration/src/main/resources/application.yaml

```diff
@@ -72,6 +72,8 @@ datahub:
     host: ${DATAHUB_GMS_HOST:localhost}
     port: ${DATAHUB_GMS_PORT:8080}
     useSSL: ${DATAHUB_GMS_USE_SSL:${GMS_USE_SSL:false}}
+    async:
+      request-timeout-ms: ${DATAHUB_GMS_ASYNC_REQUEST_TIMEOUT_MS:55000}
```

Comment:
Not directly related to dataLoaders, however this configuration does allow more then the default 30s for resolving + loading data.